modin/core/io/column_stores/parquet_dispatcher.py

```diff
@@ -60,7 +60,7 @@ class ColumnStoreDataset:
     """
 
     def __init__(self, path, storage_options):  # noqa : PR01
-        self.path = path
+        self.path = ColumnStoreDataset._get_path_str_or_handle(path)
```

Comment:
```suggestion
        self.path = path.__fspath__() if isinstance(path, os.PathLike) else path
```

What's reason to create a separate method for this?

Reply:
You raise a good point. I originally added a separate method because I thought it would be cleaner. However, I now see that is not the case and the method isn't needed anywhere either.